spring-mongo/pom.xml

```diff
@@ -30,11 +30,6 @@
       <version>${spring.boot.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.mongodb</groupId>
-      <artifactId>mongodb-driver-sync</artifactId>
-      <version>${mongo.driver.version}</version>
-    </dependency>
```

Comment:
This goal is to let `mvn dependency:tree` to display the dependency tree correctly.